samples/business-process-samples/product/sample-client/src/main/java/org/wso2/ei/businessprocess/samples/client/SamplesInvoker.java

```diff
@@ -98,11 +99,15 @@ private static void loadServiceProperties(String propFilePath, String operation)
             System.exit(0);
         } finally {
             try {
-                fin.close();
+                if (fin != null) {
+                    fin.close();
+                }
             } catch (IOException e) {

```

Comment:
Ah ... ok great. I feel adding log error is better because at least we get a notification on the error.
Anyway it's ok then.